.github/workflows/nightly_tests.yaml

```diff
@@ -66,7 +66,7 @@ jobs:
             - name: Downgrade cc and ring crate to work around https://github.com/slint-ui/slint/issues/6875
               run: |
                   cargo update -p ring --precise 0.17.9
-                  cargo update -p cc --precise 1.1.31
+                  cargo update -p cc --precise 1.2.16
```

Comment:
Do we need that at all?

Reply:
That's indeed unrelated and from another change. Oops.